gpu_bdb/queries/q10/gpu_bdb_query_10_dask_sql.py

```diff
@@ -88,9 +86,19 @@ def main(data_dir, client, c, config):
     # We extracted them from bigbenchqueriesmr.jar
     # Need to pass the absolute path for these txt files
     sentiment_dir = os.path.join(config["data_dir"], "sentiment_files")
-    ns_df = dask_cudf.read_csv(os.path.join(sentiment_dir, "negativeSentiment.txt"), names=["sentiment_word"], persist=False)
+    
+    if isinstance(product_reviews_df, dask_cudf.DataFrame):
+        ns_df = dask_cudf.read_csv(os.path.join(sentiment_dir, "negativeSentiment.txt"), names=["sentiment_word"], persist=False)
+    else:
+        ns_df = dd.read_csv(os.path.join(sentiment_dir, "negativeSentiment.txt"), names=["sentiment_word"])
```

Comment:
Why don't we have presist=False, we should remain consistent.

Reply:
The Dask `read_csv` function doesn't have a persist argument